<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>572</width>
     <height>381</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralWidget {background-image: url(:/res/main-bg.png);}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/res/main-bg.png);
background-color: rgba(207, 207, 0, 128);
</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <widget class="QWidget" name="tab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="baseSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="title">
          <string>TX</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
           <widget class="QPushButton" name="chooseFileButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>64</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background: none;
color: rgb(203, 0, 0);
background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
            </property>
            <property name="text">
             <string>File</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="sendTextButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>64</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background: none;
color: rgb(203, 0, 0);
background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
</string>
            </property>
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2" colspan="4">
           <widget class="QPlainTextEdit" name="sendTextEdit">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background: transparent;
background-color: rgba(255, 255, 255, 180);
font: 12pt &quot;Sans Serif&quot;;
</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="buttonTransmit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>64</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Sans Serif&quot;;
color: rgb(203, 0, 0);
background: none;
background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
            </property>
            <property name="text">
             <string>Voice</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkBoxTX">
              <property name="styleSheet">
               <string notr="true">QCheckBox::indicator {
     width: 32px;
     height: 32px;
}</string>
              </property>
              <property name="text">
               <string>TX</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxRX">
              <property name="styleSheet">
               <string notr="true">QCheckBox::indicator {
     width: 32px;
     height: 32px;
}</string>
              </property>
              <property name="text">
               <string>RX</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>RX</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPlainTextEdit" name="receivedTextEdit">
            <property name="styleSheet">
             <string notr="true">background: transparent;
background-color: rgb(255, 255, 255, 180);
font: 12pt &quot;Sans Serif&quot;;
</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearReceivedTextButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>64</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background: none;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));
</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tune</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>9</y>
            <width>541</width>
            <height>281</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QSlider" name="tuneSlider">
             <property name="styleSheet">
              <string notr="true">QSlider::groove:horizontal {
border: 2px solid #bbb;
background: white;
height: 120px;
border-radius: 2px;
}

QSlider::sub-page:horizontal {
border: 1px solid #777;
height: 50px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 50px;
border-radius: 4px;
}

QSlider::handle:horizontal {
border: 10px solid #777;
width: 20px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
             </property>
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Const</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Config&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <attribute name="title">
          <string>VOIP</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLineEdit" name="voipServerEdit">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="voipConnectButton">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background: none;
background-color: rgb(0, 50, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="voipDisconnectButton">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background: none;
background-color: rgb(127, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="voipTalkButton">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Sans Serif&quot;;
background:none;
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);</string>
            </property>
            <property name="text">
             <string>Talk</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="blueLED">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/res/Button-Blank-Blue-icon.png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="greenLED">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>75</x>
             <y>0</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/res/Button-Blank-Green-icon.png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="yellowLED">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>139</x>
             <y>0</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/res/Button-Blank-Yellow-icon.png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="redLED">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>204</x>
             <y>0</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/res/Button-Blank-Red-icon.png</pixmap>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearReceivedTextButton</sender>
   <signal>clicked()</signal>
   <receiver>receivedTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
